tests/test_io.py

```diff
@@ -48,7 +48,7 @@ def test_request_errors(mock_urlopen):
         # Directory structure inferred
         (
             "imap_test_l1_test-description_20100101_20100102_v00-00.cdf",
-            "test/l1/2010/01/imap_test_l1_test-description_20100101_20100102_v00-00.cdf",
+            "imap/test/l1/2010/01/imap_test_l1_test-description_20100101_20100102_v00-00.cdf",
```

Comment:
Looks like this caught an error! Do we want to put a test for the previous bad download path here as well? It might already be covered in the other file_validation tests though, so probably not a huge reason to add it here.